mobile.css

```diff
@@ -54,3 +50,104 @@
   font-weight: 600;
   color: white;
 }
+
+/* Modal  */
+.modal {
```

Comment:
- [ ] In the mobile version, please kindly correct the fonts.  You need to match the font-family, the font-weights, the line-height, the font-size, etc.  For every font on your pop-up.  Please, look at the Figma if you select an element there, the exact values will appear on the right.
- [ ] Please, kindly add the missing horizontal line between the technologies (HTML, CSS, javascript, etc) and the buttons.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/157203531-92a1decc-b36b-4cfb-b40b-7f985dfc32d9.png) | ![image](https://user-images.githubusercontent.com/57421823/157203599-25d612a6-6b6d-415f-9cb9-ceb4b3eafd1b.png)

If you implement this changes, your project will look just like the figma! 💪